### ISSUE TITLE
KEDA HTTP Addon documentation

### ISSUE THREAD

**Issue by arschles** [kedacore/keda-docs#548]
The [KEDA HTTP Addon](https://github.com/kedacore/http-add-on) project is readying to release a `v0.2.0` and we already have a significant amount of documentation. Currently, it's relatively disorganized in the [`docs/` directory](https://github.com/kedacore/http-add-on/tree/main/docs) (currently experimenting with the wiki in https://github.com/kedacore/http-add-on/pull/296) and it would be nice to have better docs that are in a dedicated space. 

Since this project is part of the KEDA organization, it seems to make sense to either be part of the `keda.sh` site or at least be in the same repository. I'm impartial to those logistics, I'd mostly just like to:

1. store documentation in a space dedicated to documentation
2. Be able to render those documentation in a logical way

### Use-Case

Write, store and render documentation in a way that's more logical to humans.

### Specification

- [ ] Move documentation here, or another place dedicated to documentation
- [ ] Render those documentation on `keda.sh`, GitBook, or another place dedicated to rendering documentation in a logical way that's friendly to humans

**Comment by tomkerkhove** [comment]
Agreed, I'm just thinking about *where*. We could have it as a "scaler" but then it has to be clear that it's not one out of the box.

**Comment by tomkerkhove** [comment]
We have a Cosmos DB external scaler as well but I'm not sure if we should "treat them as special" external scalers.

I  think HTTP is maybe a bit more exceptional scenario here, but what I have in mind is:
- We introduce "Add-ons" or "Scaler Community" which lists all external scalers from https://artifacthub.io/packages/search?kind=8&ts_query_web=keda&sort=relevance&page=1 on our website (once did POC but lost screenshot of it)
  - Stimulates the community, without us having to manage all the references
- That page has an "HTTP" subpage or link to a dedicated page for HTTP (can be dedicated in navigation, not a must)

Thoughts?

**Comment by tomkerkhove** [comment]
I'm adding this to our Documentation Improvements to see how we can further improve this

**Comment by arschles** [comment]
Sounds good @tomkerkhove. I've closed https://github.com/kedacore/http-add-on/pull/296 for now. Will keep all documentation in the `docs/` repository in the HTTP Addon repository for now until you tell me otherwise. Also I'm happy to close #554 and wait if you have bigger ideas. Let me know

**Comment by tomkerkhove** [comment]
Sounds like a good idea, but feel free to keep #554 open!

**Comment by tomkerkhove** [comment]
This should have a seperate section on our website for external scalers that are owned by KEDA org.

**Comment by arschles** [comment]
@tomkerkhove can you add the right label to prevent the stalebot from closing this? I think it's still pending some CNCF work, but I definitely don't want to close this issue meanwhile.

**Comment by Rohithzr** [comment]
+1 for it being under keda.sh

The section [Routing to the Right Service](https://github.com/kedacore/http-add-on/blob/main/docs/walkthrough.md#routing-to-the-right-service) needs better explanation of the current scenario where a scalable service gets created on the "addon-namespace" and hence the ingress needs to be created on that same namespace and be pointed to `keda-http-add-on-interceptor-proxy`

**Comment by zroubalik** [comment]
@Rohithzr that would be great!

**Comment by tomkerkhove** [comment]
@nate-double-u @thisisobate Any thoughts when this one can be planned? It has been open for a while as part of the doc improvement effort

**Comment by nate-double-u** [comment]
This issue proposes to merge KEDA add-on documentation into the main KEDA documentation, like with the Scalers. This issue mentions that we want a solution flexible enough to accommodate additional add-ons as they scale, but so far, I think there's only one such add-on (please correct me if I'm wrong here). 

I think we can close this issue, and reopen when we've got more add-ons to support.
